packages/web/src/app/onboard/page.tsx

```
@@ -1,28 +1,379 @@
+            ),
+            component: (
+                <div className="space-y-6">
+                    <div className="grid grid-cols-1 gap-3">
+                        {resourceCards.map((resourceCard) => (
+                            <a 
+                                href={resourceCard.href}
+                                target="_blank"
+                                rel="noopener"
+                                className="p-4 rounded-lg bg-accent hover:bg-accent/80 border border-border hover:border-primary/20 transition-all duration-200 group"
+                        >
+                            <div className="flex items-center gap-3">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)